packages/client/shared/tiptap/plaintextToTipTap.ts

```diff
@@ -0,0 +1,14 @@
+import {TaskTag} from '../types'
+
+export const plaintextToTipTap = (str: string, options: {taskTags?: TaskTag[]} = {}) => {
```

Comment:
building this imperatively to avoid tiptap building it with the DOM because `generateJSON` cannot be shared across server/client because it relies on a DOM